### DISCUSSION TITLE
Etcd bootstrap conflict

### DISCUSSION THREAD

**Discussion by aqeelat** [k3s-io/k3s#10365] [Q&A]
I'm trying to run two k3s clusters while sharing the same etcd. I used the suggestion in #2908, but now I get this error: `starting kubernetes: preparing server: bootstrap data already found and encrypted with different token`

What should I do? Do I use the same token between the clusters? Is there a config to make the bootstrap data also prefixed?

I'm using k3s v1.28.5+k3s1

**Comment by brandond** [comment]
> I'm trying to run two k3s clusters while sharing the same etcd. What should I do? Do I use the same token between the clusters?

You shouldn't do this. If you must, you're on your own to figure out how to make it work, because we don't support it at the moment.

That's an old comment, and the project has gotten much more integrated into etcd since then. While you might be able to get them up and running if you use the same token, other things will likely behave oddly further down the road.
